controllers/initializer/init_test.go

```diff
@@ -67,4 +67,43 @@ var _ = Describe("Init", func() {
 		})
 	})
 
+	When("initialization is called on a NHC with escalating remediation", func() {
+
+		BeforeEach(func() {
+			// create outdated default config
+			outdatedNHC := &v1alpha1.NodeHealthCheck{
+				ObjectMeta: metav1.ObjectMeta{
+					Name: defaults.DefaultCRName,
+				},
+				Spec: v1alpha1.NodeHealthCheckSpec{
+					Selector: metav1.LabelSelector{
+						MatchExpressions: []metav1.LabelSelectorRequirement{{
+							Key:      "node-role.kubernetes.io/worker",
+							Operator: metav1.LabelSelectorOpExists,
+						}},
+					},
+					EscalatingRemediations: []v1alpha1.EscalatingRemediation{
+						{
+							RemediationTemplate: corev1.ObjectReference{
+								Kind:       "PoisonPillRemediationTemplate",
```

Comment:
Nit: maybe not use Poison Pill terminology

Reply:
it's part of the test, that it does not update PP at this place 🙂